docs/ref/models/expressions.txt

```diff
@@ -18,6 +18,25 @@ Django supports negation, addition, subtraction, multiplication, division,
 modulo arithmetic, and the power operator on query expressions, using Python
 constants, variables, and even other expressions.
 
+Output field
+============
+
+Many of the expressions documented in this section allow for passing an
+optional ``output_field`` argument, into which (if given) Django will load the
+value after it's retrieved from the database.
+
+The ``output_field`` argument requires a model field instance, like
+``IntegerField()`` or ``BooleanField()``. Usually no arguments are needed when
+instantiating the model field as any arguments relating to data validation
+(``max_length``, ``max_digits``, etc.) will not be enforced on the expression's
+output value.
+
+Note that ``output_field`` is only required when Django is unable to determine
+what field type the result should be. Complex expressions that mix field types
+should define the desired ``output_field``. For example, adding an
+``IntegerField()`` and a ``FloatField()`` together should probably have
+``output_field=FloatField()`` defined.
```

Comment:
Wordsmithing...

```suggestion
Output field
============

Many of the expressions documented in this section support an optional
``output_field`` parameter. If given, Django will Django will load the
value into that field after retrieving it from the database.

``output_field`` takes a model field instance, like ``IntegerField()`` or
``BooleanField()``. Usually, the field doesn't need any arguments, like
``max_length``, since field arguments relate to data validation which will not
be performed on the expression's output value.

``output_field`` is only required when Django is unable to automatically
determine the result's field type, such as with complex expressions that mix
field types. For example, adding a ``DecimalField()`` and a ``FloatField()``
requires an output field, like ``output_field=FloatField()``.
```

It didn't sound like you were sure `IntegerField()` + `FloatField()` actually requires `output_field`. I checked the tests and found that `DecimalField()` + `FloatField()` definitely requires it: https://github.com/django/django/blob/88cb8c638cca5c2cdb5a2dd86a9c7f49ed684646/tests/aggregation/tests.py#L1130-L1139

Reply:
Thanks! Initially I grouped existing paragraphs into this new section. A review of those with fresh eyes is definitely appreciated! I applied your suggestions.